src/VirtoCommerce.CustomerExportImportModule.Data/Services/CsvPagedDataImporter.cs

```diff
@@ -111,8 +139,38 @@ public virtual async Task ImportAsync(ImportDataRequest request, Action<ImportPr
             }
         }
 
+        private void SetCountryNameToPlatformValueByIso3Code(ImportRecord<TCsvMember>[] dataSourceItems)
+        {
+            foreach (var importRecord in dataSourceItems)
+            {
+                var countryCode = importRecord.Record.AddressCountryCode;
+
+                if (!string.IsNullOrEmpty(countryCode) && countryCode.Length == 3)
```

Comment:
Here and below checks for null is probably redundant, because CountryCode already validated

Reply:
This processing took place before validation.